.github/workflows/production-cluster-check.yml

```diff
@@ -67,7 +67,7 @@ jobs:
           SLACK_CHANNEL_SECOND: ${{ secrets.SLACK_CHANNEL_SECOND }}
           WEBHOOK_SITE_TOKEN: ${{ secrets.WEBHOOK_SITE_TOKEN }}
           WEBHOOK_SITE_TOKEN_SECOND: ${{ secrets.WEBHOOK_SITE_TOKEN_SECOND }}
-      - name: Run integration tests
```

Comment:
Production cluster CI should be updated once production environment is updated.
Not sure if the update will be done in same time or delayed? If delayed then this change should be removed and applied after production update.

Reply:
Good point, I guess it is better to change the production environment after the change has been validated on the other envs, so I will remove it.

Reply:
Reverted the change for production cluster CI check.